src/gerbil/core/expander.ss

```diff
@@ -41,6 +41,7 @@ package: gerbil/core
     current-expander-path
     current-expander-phi
     current-expander-compiling?
+    current-compilation-target?
```

Comment:
why the `?`?

It should be just current-compilation-target.

Reply:
Typo, which is why I added the gx# prefix FFS. Doh!! My mistake.